<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridTab</class>
 <widget class="QWidget" name="GridTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="_showGrid">
     <property name="text">
      <string>Show &amp;grid</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QCheckBox" name="_snapToGrid">
     <property name="text">
      <string>&amp;Snap to grid</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Grid &amp;horizontal spacing:</string>
     </property>
     <property name="buddy">
      <cstring>_gridHorSpacing</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="_gridHorSpacing"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Grid &amp;vertical spacing:</string>
     </property>
     <property name="buddy">
      <cstring>_gridVerSpacing</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="_gridVerSpacing"/>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
